### DISCUSSION TITLE
Issue with concentration dependent transference number

### DISCUSSION THREAD

**Discussion by MarcAntoine88** [pybamm-team/PyBaMM#2681] [Q&A]
Hello all, 

When I run the DFN model with ORegan2022 parameter set, the concentration of Li in the electrolyte shows an increase as I discharge the cell (the area under the curve is not constant) which is not normal to me. 

This only happens when the transference number depends on the concentration in the electrolyte.
Can someone help ? 
Thanks a lot, 

Marc-Antoine
![ORegan_after_139_sec](https://user-images.githubusercontent.com/94439193/218445802-275d0bb0-6bb3-4804-bc2e-a8905cd32700.png)
![ORegan_after_1800_sec](https://user-images.githubusercontent.com/94439193/218445803-b0bf55b4-e00f-4cda-aa71-01af561ebb1e.png)

**Comment by brosaplanella** [comment]
Hi! I think this is related to issue #2666. Can you double check, and if so we can continue the discussion there?

**Reply by MarcAntoine88** [reply]
Hi @brosaplanella, thanks for the quick feedback, yes this is the same issue! Sorry I didn't see it before opening this discussion.

**Reply by brosaplanella** [reply]
No worries, I have copied the images and we can continue the discussion there.
